app/controllers/teams_controller.rb

```diff
@@ -82,6 +82,23 @@ def invite
     end
   end
 
+  def looking_for_members
```

Comment:
This functionality can just be placed on the team edit page instead of having to have its own method.